src/administration/Administration.Service/Controllers/StaticDataController.cs

```diff
@@ -104,4 +104,19 @@ public StaticDataController(IStaticDataBusinessLogic logic)
     [ProducesResponseType(typeof(IAsyncEnumerable<OperatorBpnData>), StatusCodes.Status200OK)]
     public IAsyncEnumerable<OperatorBpnData> GetOperatorBpns() =>
         _logic.GetOperatorBpns();
+
+    /// <summary>
+    /// Retrieve all certificate types 
+    /// </summary>
+    /// <returns>AsyncEnumerable of certificate types</returns>
+    /// <remarks>
+    /// Example: GET: /api/administration/staticdata/certificateTypes    
+    /// </remarks>
+    /// <response code="200">Returns a list of all of the Language i.e german and english</response>
+    [HttpGet]
+    [Authorize(Roles = "view_certificates")]
+    [Route("certificateTypes")]
+    [ProducesResponseType(typeof(IAsyncEnumerable<CompanyCertificateTypeData>), StatusCodes.Status200OK)]
```

Comment:
Please change to IEnumerable, Swagger UI has problems with displaying IAsyncEnumerable as a ResponseType. 

```suggestion
    [ProducesResponseType(typeof(IEnumerable<CompanyCertificateTypeData>), StatusCodes.Status200OK)]
```

Reply:
replaced with IEnumerable